float64[] destination_2
float64 v_2
float64 a_2
bool io_0_2